Dockerfile

```diff
@@ -17,28 +28,16 @@ LABEL "org.opencontainers.image.authors"="Ryan Eberhard <ryan.eberhard@oracle.co
       "org.opencontainers.image.description"="Oracle WebLogic Server Kubernetes Operator" \
       "org.opencontainers.image.documentation"="https://oracle.github.io/weblogic-kubernetes-operator/"
 
-RUN set -eux; \
-    microdnf -y install gzip tar openssl jq; \
-    microdnf -y update; \
-    microdnf clean all
+ENV LANG="en_US.UTF-8"
 
-ENV LANG="en_US.UTF-8" \
-    JAVA_HOME="/usr/local/java" \
-    PATH="/operator:$JAVA_HOME/bin:$PATH" \
-    JAVA_URL="https://download.java.net/java/GA/jdk17.0.2/dfd4a8d0985749f896bed50d7138ee7f/8/GPL/openjdk-17.0.2_linux-x64_bin.tar.gz"
+COPY --from=jre-build /jre jre
 
 # Install Java and make the operator run with a non-root user id (1000 is the `oracle` user)
 RUN set -eux; \
-    curl -fL -o /jdk.tar.gz "$JAVA_URL"; \
-    mkdir -p "$JAVA_HOME"; \
-    tar --extract --file /jdk.tar.gz --directory "$JAVA_HOME" --strip-components 1; \
-    rm /jdk.tar.gz; \
-    mkdir /usr/java; \
-    ln -sfT "$JAVA_HOME" /usr/java/default; \
-    ln -sfT "$JAVA_HOME" /usr/java/latest; \
-    rm -Rf "$JAVA_HOME/include" "$JAVA_HOME/jmods"; \
-    rm -f "$JAVA_HOME/lib/src.zip"; \
-    for bin in "$JAVA_HOME/bin/"*; do \
+    microdnf -y install jq; \
+    microdnf -y update; \
```

Comment:
Are there any reasons not to update repo before install?

Reply:
I don't think it matters. The install will install the latest version of jq. The update call is to make sure that other components, especially system components are at latest. We were asked to add this because we were told that the dnf repositories are updated prior to new container images being published.